src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/model/out/reports/EnvelopeCountSummaryReportListResponse.java

```diff
@@ -1,15 +1,49 @@
 package uk.gov.hmcts.reform.bulkscanprocessor.model.out.reports;
 
+import com.fasterxml.jackson.annotation.JsonFormat;
 import com.fasterxml.jackson.annotation.JsonProperty;
 
+import java.time.LocalDateTime;
 import java.util.List;
 
 public class EnvelopeCountSummaryReportListResponse {
 
+    @JsonProperty("total_received")
+    private final int totalReceived;
+
+    @JsonProperty("total_rejected")
+    private final int totalRejected;
```

Comment:
Other class attributes are public, make these 2 also public, in this case getters for them will not be necessary.

Reply:
fixed